### ISSUE TITLE
Reader Document Viewer: Page loses visual indication of focus after tabbing through `Search cases`

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12245]
When a document is selected to be displayed in the reader, there is no visual indication of focus after the Search Text element. (New: 08/27/2019) I used my tool to show the focus in yellow below however you cannot visually see this focus without the tool on.  This loss of focus happens for seven tab stops.

<img width="1310" alt="Screen Shot 2019-09-27 at 5 18 59 PM" src="https://user-images.githubusercontent.com/1198106/65802887-ea862480-e14a-11e9-8ceb-98e973986bee.png">

**Comment by ajspotts** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | |||
3 | |||||||||
5 | 
8 | 
```



#### Why 1?
* 

#### Why 2?
* Optimism?
* This sounds straightforward once we get our bearings
* Looks like there are a few unseen elements that are being selected.

#### Why 3?
* General lack of knowledge around how active elements are determined and displayed

**Comment by yoomlam** [comment]
The initially hidden search bar `cf-search-bar` constitutes 4 tab stops:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/55255674/84708919-f73fe800-af26-11ea-90c8-63fb518da5a7.png">

The `cf-pdf-scroll-view` requires 3 tab stops (one for each `ReactVirtualized__Grid`):
<img width="550" alt="image" src="https://user-images.githubusercontent.com/55255674/84709229-77664d80-af27-11ea-82ca-399c9618979c.png">
